recipes/boost-wintls/all/conanfile.py

```diff
@@ -0,0 +1,91 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.layout import basic_layout
+from conan.tools.files import copy, get
+from conan.tools.scm import Version
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class BoostWinTLS(ConanFile):
+    name = "boost-wintls"
+    description = "Native Windows TLS stream wrapper for use with boost::asio"
+    license = "BSL-1.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://wintls.dev"
+    topics = (
+        "header-only",
+        "windows",
+        "tls",
+        "ssl",
+        "networking",
+        "boost",
+        "asio",
+        "sspi",
+        "schannel",
+    )
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+
+    @property
+    def _min_cppstd(self):
+        return 14
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "clang": "12",
```

Comment:
```suggestion
            "clang": "7",
            "apple-clang": "10",
```

The Clang 7 supports C++14 already: https://en.cppreference.com/w/cpp/compiler_support/14

Apple-Clang 10 supports C++14 too: https://developer.apple.com/xcode/cpp/#c++14